### DISCUSSION TITLE
Any way to permanently disable wifi-connect?

### DISCUSSION THREAD

**Discussion by ghost** [AlexProgrammerDE/balena-minecraft-server#142] [Q&A]
wifi-connect makes the Raspberry Pi 4 have the ability to be connected to directly via wifi. It keeps turning itself on after reboot.

**Comment by viedok** [comment]
I need that too ! Alex if you are reading this please help :')

**Reply by AlexProgrammerDE** [reply]
Hi! This is as easy as removing line 32-39 from the compose file and redeploying the project!
https://github.com/AlexProgrammerDE/balena-minecraft-server/blob/60a968e5cacdbc14359991829cbbacf22a552a93/docker-compose.yml#L32-L39
